### PR TITLE
google place api key sent from the view for survey 3

### PR DESCRIPTION
**Pull request by hsklogix**

### CODE REVIEW

File: lms/djangoapps/onboarding_survey/views.py [hunk 1]
```diff
@@ -284,6 +284,7 @@ def organization(request):
     context['is_poc'] = extended_profile.is_poc
     context['is_first_user'] = is_first_signup_in_org(extended_profile.organization)
     context['organization_name'] = extended_profile.organization.name
+    context['google_place_api_key'] = settings.GOOGLE_PLACE_API_KEY
```
Comment (muhammadnadeem) [inline lms/djangoapps/onboarding_survey/views.py]: Why are you sending it via context. Are we able to get GOOGLE_PLACE_API_KEY directly from settings inside template?

Comment (hsklogix) [reply lms/djangoapps/onboarding_survey/views.py]: It's a django template not mako and we cannot access settings inside that template directly.

Comment (muhammadnadeem) [reply lms/djangoapps/onboarding_survey/views.py]: ok that's fine then

**Comment by muhammadnadeem** [comment]
LGTM

**Pull request merged**